### DISCUSSION TITLE
Will CMI support EN 2.5 version?

### DISCUSSION THREAD

**Discussion by aryoadhi** [krypto5863/COM-Modular-Installer#17] [General]
The title says it all. I've read the documentation, and unsure if it's even supported. All I can see is the installer should've able to detect JP, JP with 2.5, and EN. EN with 2.5 is not listed though, even though it's officially released...

**Comment by krypto5863** [comment]
I did a quick review of my code and it _seems_ to be all good and accounted for. You'll know it's okay if you get two different warnings. One telling you it's English and another telling you it's 2.5. If you see both of these, go with the wind and close the comment, otherwise let me know and I'll see what can be done.

**Reply by aryoadhi** [reply]
I'll see about that. Just in case something goes wrong I'll uninstall existing mods first before reinstalling CMI.

**Comment by aryoadhi** [comment]
After some further testing, I can simply install these mods. I might open up an issue when something goes wrong with the CMI installer.
